tools/pontoon/translate.md

```diff
@@ -0,0 +1,82 @@
+# Translating in Pontoon
+
+* [Translating Strings](#translating-strings)
+  * [Submit a Suggestion](#submit-a-suggestion)
+  * [Review a Suggestion](#review-a-suggestion)
+  * [Submit a Translation](#submit-a-translation-directly)
+* [Translating in the In-context Editor](#translating-in-the-wysiwyg-editor)
+
+## Translating Strings
+
+Three string actions can be performed: submit a suggestion, review a suggestion, and submit a translation directly. The level to which a user can translate a string is dependent upon their user role (Read about user roles [here](user.md)).
+
+*This documentation is simplified to demonstrate translation modes in Pontoon. Click [here](ui.md) to learn how to use the Pontoon interface and [here](resources.md) to learn about Pontoon's translation resources.*
+
+### Submit a Suggestion
+
+When a translator is in Suggest Mode, the blue 'Suggest' button will be located to the lower-right side of the editing space.
+
+![Blue button for suggestions](/assets/images/pontoon/translate/suggest_button.png)
+
+To suggest a translation, the translator should simply input the translation to the editing space and click `Suggest`. Once the translator has suggested the translation, the string's status will be updated from **missing** to **suggested**. This is indicated in the sidebar by the color of the checkbox next to the string.
+
+![Checkbox before suggestion](/assets/images/pontoon/translate/sidebar_presuggest.png) ![Checkbox after suggestion](/assets/images/pontoon/translate/sidebar_postsuggest.png)
+
+A notification will appear at the top of the window, indicating that your translation has been saved.
+
+![Translation saved notification](/assets/images/pontoon/translate/translation_saved_notif.png)
+
+*Note that a suggested translation exists only in the Pontoon database and not in the localized file or translation memories.*
+
+#### Manually Switch to Suggestion Mode
+
+To manually switch to Suggestion Mode, click the Settings gear to the lower-left side of the editing space and click the icon to the left of 'Make Suggestions' so that it becomes a green circle. The `Suggest` button should then appear to the lower-right side of the editing space.
```

Comment:
Worth adding something like "If you’re a translator, you can still decide to provide suggestions:"?